ls1intum/Artemis
# `Programming exercises`: Fix editing of exam exercises

### Checklist
#### General


#### Client

### Motivation and Context
Editing exam exercises was showing input for release and due dates, which are not valid for an exam.

### Description
Now the values are hidden when editing an exam exercise.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Exam Programming Exercise

1. Edit the programming exercise
2. Verify that no due date input fields are shown and also manual feedback requests are not activateable.
3. Switch to the wizzard mode and check that no due date fields are shown here as well

### Review Progress

#### Code Review
#### Manual Tests

### Screenshots
before:
![grafik](https://user-images.githubusercontent.com/26540346/233578338-390141e3-039b-476e-aea7-3dc58a15c712.png)


now:
![grafik](https://user-images.githubusercontent.com/26540346/233578217-ae36e28c-e299-4d09-8810-d5e6acf86947.png)
